docs/toolchains/raspberry-pi.rst

```diff
@@ -0,0 +1,38 @@
+.. Copyright (c) 2017, Ruslan Baratov
+.. All rights reserved.
+
+Raspberry Pi
+------------
+
+.. seealso::
+
+  * `Official <https://www.raspberrypi.org/>`__
+  * `Hardware Guide <https://www.raspberrypi.org/learning/hardware-guide/>`__
+  * `Formatting SDCard <https://www.raspberrypi.org/documentation/installation/sdxc_formatting.md>`__
+  * `Download Software <https://www.raspberrypi.org/downloads/noobs/>`__
+  * `What password to use to log in after the first boot? <https://raspberrypi.stackexchange.com/q/660/70510>`__
+
+Instructions for `Raspbian <https://www.raspberrypi.org/downloads/raspbian/>`__:
+
+.. code-block:: none
+  :emphasize-lines: 3
+
+  > lsb_release -a
+  No LSB modules are available.
+  Distributor ID: Raspbian
+  Description:    Raspbian GNU/Linux 8.0 (jessie)
+  Release:        8.0
+  Codename:       jessie
+
+
+.. code-block:: none
+
+  > sudo apt-get install python3
+  > sudo apt-get install g++
+  > sudo apt-get install cmake
+
+Use ``gcc`` toolchain, e.g.:
+
+.. code-block:: none
+
+  > polly.py --toolchain gcc --verbose --config Release
```

Comment:
> Yep, cross compiling

@pretyman Can't get it work. I should use this repository with tools: https://github.com/raspberrypi/tools, right? I think there is problem with flags defined here:
* https://github.com/ruslo/polly/blob/f7dd93ab12d0b9e20dcf1146c1c90ce8b836272b/os/raspberry-pi3.cmake#L12
* https://github.com/ruslo/polly/blob/f7dd93ab12d0b9e20dcf1146c1c90ce8b836272b/os/raspberry-pi-hardfloat.cmake#L12

Where they came from?